src/background/stores/assessment-store.ts

```diff
@@ -147,6 +149,10 @@ export class AssessmentStore extends BaseStoreImpl<AssessmentStoreData> {
         this.updateTargetTabWithId(tabId);
     };
 
+    private onUploadAssessment = (payload: UploadAssessmentPayload): void => {
+        //update state
```

Comment:
Dan had mentioned on a call including a ADO work item number with this comment referencing a user story or feature tracking this work. That could be helpful!